lib/ext/heroku/command/addons.rb

```diff
@@ -225,6 +225,14 @@ def add
       end
     end
 
+    # addons:add ADDON
+    #
+    # WARNING: `heroku addons:add has been deprecated. Please use `heroku addons:create` instead.
```

Comment:
@bjeanes hmm I'll give it a shot without the comments being there; I assumed it was something of a convention as I found this style in another deprecated method (in the mainline Toolbelt codebase)